docs/UPGRADE.md

```diff
@@ -0,0 +1,39 @@
+# Upgrade
+
+## From 2.0.0
+
+A new table has been added to the table `ecchronos.on_demand_repair_status`, this must be added before upgrading.
+
+The command to add the column is shown below:
+```
+ALTER TABLE ecchronos.on_demand_repair_status ADD completed_time timestamp;
+```
+
+Note: Be careful and make sure that you create the column with the correct cql_type since its not possible to change cql_type on an existing column.
```

Comment:
Make sure that the column is created with the correct cql_type since it is not possible to change cql_type on an existing column.